app/index.js

```diff
@@ -363,7 +362,9 @@ app.on('ready', () => {
 
     AppStore.addChangeListener(() => {
       Menu.init(AppStore.getState().get('settings'))
-      initiateSessionStateSave()
+      // This is debounced to every 5 minutes, the save is not particularly intensive but it does do IO
```

Comment:
oops sorry, for some reason I thought it was changed to 1 minute but it's clearly still 5.